copier.yaml

```diff
@@ -113,3 +129,52 @@ default_py:
       value: '3.11'
   help: 'Choose the default Python version for development, documentation generation, and package build:'
   type: str
+
+copyright_license:
+  type: str
+  help: Your project's license. If you have no idea how to choose one, please see https://choosealicense.com/.
+  default: MIT License
+  choices:
+    - Academic Free License v3.0
+    - Apache License 2.0
+    - Artistic License 2.0
+    - BSD 2-Clause "Simplified" License
+    - BSD 3-Clause Clear License
+    - BSD 3-Clause "New" or "Revised" License
+    - Boost Software License 1.0
+    - Creative Commons Attribution 4.0
+    - Creative Commons Attribution Share Alike 4.0
+    - Creative Commons Zero v1.0 Universal
+    - Do What The F*ck You Want To Public License
+    - Educational Community License v2.0
+    - Eclipse Public License 1.0
+    - Eclipse Public License 2.0
+    - European Union Public License 1.1
+    - European Union Public License 1.2
+    - GNU Affero General Public License v3.0
+    - GNU General Public License v2.0
+    - GNU General Public License v3.0
+    - GNU Lesser General Public License v2.1
+    - GNU Lesser General Public License v3.0
+    - ISC License
+    - LaTeX Project Public License v1.3c
+    - MIT License
+    - Mozilla Public License 2.0
+    - Microsoft Public License
+    - Microsoft Reciprocal License
+    - University of Illinois/NCSA Open Source License
+    - SIL Open Font License 1.1
+    - Open Software License 3.0
+    - PostgreSQL License
+    - The Unlicense
+    - zlib License
+
+copyright_date:
+  type: str
+  help: The copyright date
+  default: '2022-2023'
+
+copyright_holder:
+  type: str
+  help: The name of the person/entity holding the copyright
+  default: '{{ author_name }}'
```

Comment:
Let us use the `organization_name` instead. Feel free to change the corresponding LICENSE file.